teachers_digital_platform/jinja2/teachers_digital_platform/activity_page.html

```diff
@@ -62,6 +62,15 @@ <h3 class="h5">Student materials</h3>
                 </p>
             {% endif %}
     </div>
+    <div>
+        <h2 class="u-mt45">Explore related resources</h2>
+        <ul>
+            {% set parent_page = page.get_parent().url %}
```

Comment:
Wagtail recommended convention is to use the `pageurl` template tag instead of accessing `.url` directly:

```suggestion
            {% set parent_page = pageurl(page.get_parent()) %}
```